search/routes/api/classic.py

```diff
@@ -28,18 +28,16 @@ def query() -> Response:
     # requested = request.accept_mimetypes.best_match([JSON, ATOM_XML])
     # if requested == ATOM_XML:
     #     return serialize.as_atom(data), status, headers
-    response = serialize.as_json(data['results'], query=data['query'])
-    response.status_code = status_code
-    response.headers.extend(headers)
-    return response
+    response_data = serialize.as_atom(data['results'], query=data['query'])
+    return response_data, status_code, headers # type: ignore
```

Comment:
This looks like a regression. The return type is ``flask.Response``, so the former code was correct (at least with respect to typing). Did you run into a problem here?